gradle.properties

```diff
@@ -24,4 +24,4 @@ systemProp.knownSparkVersions=2.4,3.0,3.1,3.2,3.3
 systemProp.defaultScalaVersion=2.12
 systemProp.knownScalaVersions=2.12,2.13
 org.gradle.parallel=true
-org.gradle.jvmargs=-Xmx768m
+org.gradle.jvmargs=-Xmx1024m
```

Comment:
it seems that the plugin needs slightly more memory as it was running OOM with 768M